### ISSUE TITLE
Parsing error on Vue template filter with colon (luxon:locale)

### ISSUE THREAD

**Issue by IlyaSemenov** [vuejs/eslint-plugin-vue#687]
eslint-plugin-vue wouldn't parse vue template filters with a colon in it, e.g. `luxon:locale` (which is supposedly legitimate syntax: https://github.com/casbloem/vue-luxon#luxonlocale).

<img width="699" alt="screenshot 2018-11-29 at 14 33 28" src="https://user-images.githubusercontent.com/128121/49206161-c9965b00-f3e3-11e8-9d84-bf05b4a61880.png">

**Tell us about your environment**
* **ESLint version:**	4.19.1
* **eslint-plugin-vue version:**	4.5.0
* **Node version:**		10.12.0

**Please show your full configuration:**

```js
module.exports = {
  root: true,
  extends: [
    'plugin:vue/essential',
    '@vue/standard'
  ]
}
```

**What did you do?**	

```vue
<template>
  <div>{{ user.create_time | luxon:locale('short') }}</div>
</template>
```

**What did you expect to happen?**

No errors.

**What actually happened?**

```
/Users/semenov/xxx/test.vue
  2:35  error  Parsing error: Unexpected token :  vue/no-parsing-error

✖ 1 problem (1 error, 0 warnings)
```

**Comment by mysticatea** [comment]
Thank you for this issue.

JavaScript doesn't have such a syntax, so `espree`, a JavaScript parser, throws a syntax error on that `:`.

@yyx990803 would you know how that works? I couldn't find that syntax in [Filters](https://vuejs.org/v2/guide/filters.html) page.

**Comment by michalsnik** [comment]
@mysticatea Filters in Vue are parsed and put on stack as strings. So in this case, the following expression:
```
{{ user.create_time | luxon:locale('short') }}
```

Generates the following function:
```
_f("luxon:locale")(user.create_time,'short')
```

So we should probably also treat the filter part in our parser as a plain string I guess as it doesn't have to conform to espree.

**Comment by mysticatea** [comment]
Thank you!

Hmm, it's tough...

**Comment by mysticatea** [comment]
I will take a look in this weekend.
